ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -1234,4 +1214,19 @@ private class PasteControlTextField: UITextField {
         isPaste = true
         super.paste(sender)
     }
+
+    override func deleteBackward() {
+        // Triggers the delegate method even when the cursor (caret) is in the first postion (regardless of text being empty).
+        // Using the zero width space ("\u{200B}") as the emptyTextFieldString instead of this approach will cause Voice Over
+        // to read it out loud as "zero width space", which is not desirable.
+        if let selectionRange = selectedTextRange,
+           selectionRange.isEmpty,
+           offset(from: beginningOfDocument, to: selectionRange.start) == 0 {
+            _ = self.delegate?.textField?(self,
+                                          shouldChangeCharactersIn: NSRange(location: 0, length: 0),
+                                          replacementString: "")
```

Comment:
use `emptyTextFieldString` instead of `""` ?

Reply:
It's a private type of BadgeField. Probably not worth exposing it just for that constant.